src/Components/Archive/ArchivePage.jsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
+import { useArchive, useNotes, useTrash } from '../../context/';
 
 const ArchivePage = () => {
-	return <div>ArchivePage</div>;
+	const { archiveNotes, deleteArchiveNotes, archiveToNotes } = useArchive();
+	const { archiveToTrash } = useTrash();
+	const { getNotes } = useNotes();
+	return (
+		<div className='app-content'>
+			{archiveNotes.length > 0 ? (
+				<div>
+					{archiveNotes.map((note) => {
+						return (
+							<div className='card ' key={note._id}>
```

Comment:
Suggestion: extract this as a note component as you will be reusing it multiple times right?

Reply:
This will be done in the next PR.